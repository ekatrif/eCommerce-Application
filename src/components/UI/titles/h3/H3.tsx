import React from 'react';
import classes from './H3.module.css';

interface H3Props {
  text?: string;
}

const H3: React.FC<H3Props> = ({ text }) => {
  return <h3 className={classes.h3}>{text}</h3>;
};

export default H3;
