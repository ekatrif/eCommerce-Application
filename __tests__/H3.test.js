import React from 'react';
import renderer from 'react-test-renderer';
import H3 from '../src/components/UI/titles/H3/H3';

test('H3 title renders correctly', () => {
  const tree = renderer
    .create(<H3 text="h3 title text" />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
