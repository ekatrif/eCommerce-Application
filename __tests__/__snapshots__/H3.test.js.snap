// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`H3 title renders correctly 1`] = `
<h3
  className="h3"
>
  h3 title text
</h3>
`;
